### PR TITLE
add styling to-description page

### LINKED ISSUE #60: create description page with css
Relates #45 
Add styles to the description page to make it looks like Figma design

### PR DESCRIPTION
**Pull request by mo7amedehab97**
add some classes to the description.html page and give the page styling relates #60

### CODE REVIEW

File: Description/Description.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+.header{
+    position: relative;
+}
+.headerBtn{
+ position: absolute;
+ top: 8%;
```
Comment (EmanHass) [inline Description/Description.css:11]: set top=3%

File: Description/Description.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+.back-icon{
+    font-size: 25px;
+    position: absolute;
+    left: 25px;
+}
+.cart-icon{
+    font-size: 25px;
+
+    position: absolute;
+    right: -375px;
```
Comment (EmanHass) [inline Description/Description.css:23]: set right:-331px

File: Description/Description.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    z-index: 1;
+}
+
+div img{
+max-width: 100%;
```
Comment (EmanHass) [inline Description/Description.css:28]: max-width:91% to remove scroll
height:48vh

**Review by EmanHass — CHANGES_REQUESTED** [review]

**Review by MMansour87 — DISMISSED** [review]

**Review by EmanHass — DISMISSED** [review]

**Review by MMansour87 — DISMISSED** [review]
Well Done

**Review by abdulrahman-2020 — DISMISSED** [review]

**Pull request merged**